testsuite/src/it/java/org/graalvm/tests/integration/utils/Commands.java

```diff
@@ -1270,4 +1282,16 @@ public static void cleanup(Process process, String cn, String mn, StringBuilder
         Logs.writeReport(cn, mn, report.toString());
         cleanTarget(app);
     }
+
+    public static boolean compareArrays(int[] a, int[] b, int[] threshold) {
+        if (a.length != b.length || a.length != threshold.length) {
+            return false;
+        }
+        for (int i = 0; i < a.length; i++) {
+            if (Math.max(a[i], b[i]) - Math.min(a[i], b[i]) > threshold[i]) {
```

Comment:
Minor: I find 
```suggestion
            if (Math.abs(a[i] - b[i]) > threshold[i]) {
```
easier to read.

Reply:
It's not only easier to read, but also the correct use of the function :-)   thx.

Reply:
resolevd in [5040e72](https://github.com/Karm/mandrel-integration-tests/pull/313/commits/5040e724020803187ea9db4592f6e93d883e78b5)